api_test/src/test/java/ecse429/group7/TestTodos.java

```diff
@@ -0,0 +1,320 @@
+package ecse429.group7;
+
+import ecse429.group7.BaseTest;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.junit.BeforeClass;
+
+import kong.unirest.Unirest;
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+
+
+public class TestTodos extends BaseTest
+{
+
+    // GET /todos
+    @Test
+    public void testGetTodosStatusCode()
+    {
+        assertGetStatusCode("/todos", STATUS_CODE_OK);
+    }
+
+    @Test
+    public void testGetTodosResponseSize()
+    {
+        HttpResponse<JsonNode> response = Unirest.get("/todos").asJson();
+        assertEquals(response.getBody().getObject().getJSONArray("todos").length(), 2);
+    }
+
+    @Test
+    public void testGetTodosResponseFirstTitle()
```

Comment:
Should we make the assumption that the first title is named "file paperwork"? An alternative is to use either a before test or before all tests to set up an initial state.

I personally think this is fine.

Reply:
Probably not a good assumption, will change